### ISSUE TITLE
Crash while sleeping, also couldn't see while lying in bed

### ISSUE THREAD

**Issue by Darkmega18** [Su5eD/Somnia-Awoken#34]
**Describe the bug**  
I slept in bed. it crashed after a while while the meter which appeared near the top ticked up towards the end.

**Is this bug related to another mod?**  
Quite possibly, it's a concurrent modification. But I have no idea what to point the finger at since it only lists your stuff in the error.

**To Reproduce**  
I guess you'd have to play my pack to have it happen, cause I'm otherwise clueless. :V but I literally just slept for the first time after reaching 95+ fatigue, then when I laid in bed, I couldn't see anything other than the hue of the sky then it crashed as the meter filled up.

**Expected behavior**  
.... to be able to sleep and it work properly and reduce my fatigue? Which it did after I loaded back in after returning to main menu, but I was hesitant to let it build up towards full again.

**Screenshots**  
it's like I wedged under the bed or something or fell inside it.
![image](https://user-images.githubusercontent.com/8104689/109993093-0d653100-7d60-11eb-9ffc-f23ceb029227.png)

crash report is attached, and you can see the quite extensive list of mods and all your required versions and stuff. :V 
[crash-2021-03-05_03.02.29-client.txt](https://github.com/Su5eD/Somnia/files/6085019/crash-2021-03-05_03.02.29-client.txt)

**Comment by Darkmega18** [comment]
I can't seem to make it happen again though after doing a segmented full day sleep though, so it's odd.

**Comment by Su5eD** [comment]
Regarding the vision problem, do you have `vanillaBugFixes` or `disableRendering` enabled?

**Comment by Darkmega18** [comment]
I turned off vanillabugfixes just now, and disable rendering was off already but I haven't gotten sleepy enough to rest just yet and I gotta go sleep irl cause got stuff on tomorrow. but I suspected it might've been related to those settings, yes. I'll check it next time I keep testing things. Since I tweaked the fatigue rate around and stuff like that. as for the concurrent modification crash, not sure what caused it, since I was able to sleep the rest of the fatigue off just fine with a bunch of furnaces and pulvuses going over the next day and night in game.
